docs/init.md

```diff
@@ -77,7 +77,7 @@ module.exports = {
   templateDir: './template',
 
   // Path to script, which will be executed after initialization process, but before installing all the dependencies specified in the template.
-  postInitScript: './script.js',
+  postInitScript: './script.sh',
```

Comment:
Nah, let's use JS. You can make your `.js` file executable using shebang: https://github.com/Esemesek/react-native-new-template/blob/master/script.js
or update this file to:
```
postInitScript: 'node ./script.js'
```

Reply:
I tried `node ./script.js` but that gave me an error. I'm fine with adding a shebang in the script, but I think it makes sense to document that somewhere at least.

Reply:
Ah sorry, I totally oversaw that you added the shebang already in the sample script.

Reply:
I'm fine with making it more obvious though :)